### PR TITLE
Sentinel incident number fix

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
- [x] Ready

## Related Issues
fixes: [XSUP-36933](https://jira-dc.paloaltonetworks.com/browse/XSUP-36933)

## Description
The field is now searchable from the incidents page.

### CODE REVIEW

File: Packs/AzureSentinel/ReleaseNotes/1_5_41.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Incident Fields
+
+- **Microsoft Sentinel Incident Number**
```
Comment (ShahafBenYakir) [inline Packs/AzureSentinel/ReleaseNotes/1_5_41.md:4]: ?

**Review by ShahafBenYakir — COMMENTED** [review]
Have you synced with the security team? I think this was done on purpose

File: Packs/AzureSentinel/ReleaseNotes/1_5_41.md (hunk 1 above)
Comment (Shellyber) [reply Packs/AzureSentinel/ReleaseNotes/1_5_41.md]: It was the default - I updated it

**Comment by Shellyber** [comment]
@ShahafBenYakir I confirmed it with Meital from the security team. They don't design this. I think it is the default when creating incident field. 
If it does not break BC then it is fine to change it.

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**